lasertagger/custom_utils.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License
+
+"""Customized utils for modified LaserTagger model."""
+
+import json
+import os
+import nltk
+
+from typing import Text, List
+nltk.download('averaged_perceptron_tagger')
```

Comment:
I remember you can do this in a __init__.py file under the same dir.

I have never seen any other code inside Google directly calling a function in the global scope.

Reply:
Done